modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/util/DDMFormFieldTypeUtil.java

```diff
@@ -24,15 +24,64 @@
 import com.liferay.portal.kernel.log.Log;
 import com.liferay.portal.kernel.log.LogFactoryUtil;
 import com.liferay.portal.kernel.util.GetterUtil;
+import com.liferay.portal.kernel.util.MapUtil;
 
 import java.util.Locale;
+import java.util.Map;
 import java.util.stream.Stream;
 
 /**
  * @author Marcela Cunha
  */
 public class DDMFormFieldTypeUtil {
 
+	public static String getPropertyValue(
```

Comment:
Yes. I was avoiding duplicate code in `ValidationDDMFormFieldTemplateContextContributor`. 
I need all the parameters related to the `InputMask` feature to fix the bug in the Validation field.  For example, this method is used to get the value of `numericInputMask`. Therefore, I thought it would be better to move everything to this Util class which already has methods similar to this one.

Reply:
It makes sense. I'm just concerned adding two more methods named `getPropertyValue` is going to get confusing and hard to figure out which one you need. Perhaps these should be called `getChangedPropertyValue` since the code is retrieving from the `changedProperties` map that resides in the `properties` map.
Also, what do you think of putting this first method, the one that's only used in the `NumericDDMFormFieldTypeUtil`, inside the `NumericDDMFormFieldTypeUtil`?

Reply:
Yeah, I also had this concern.

> Also, what do you think of putting this first method, the one that's only used in the NumericDDMFormFieldTypeUtil, inside the NumericDDMFormFieldTypeUtil?

Great idea! I'll make the changes!